docs/shared/packages/cypress/cypress-plugin.md

```diff
@@ -97,7 +97,9 @@ export default defineConfig({
 });
 ```
 
-Note: The `nxE2EPreset` is a collection of default settings, but is not necessary for task splitting.
```

Comment:
This turns out to be not true since local and CI runs depend on `webServerCommands` and `ciWebServerCommand` to start and listen to server ready. There are ways users can customize the startup on their own, but by default we want them to use our web server support.